paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/customer/CustomerSheetViewModel.kt

```diff
@@ -7,15 +7,15 @@ import com.github.kittinunf.fuel.Fuel
 import com.github.kittinunf.fuel.core.FuelError
 import com.github.kittinunf.fuel.core.extensions.jsonBody
 import com.github.kittinunf.fuel.core.requests.suspendable
-import com.stripe.android.ExperimentalCustomerSheetApi
 import com.stripe.android.PaymentConfiguration
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import com.stripe.android.customersheet.repositories.CustomerAdapter
```

Comment:
```suggestion
import com.stripe.android.customersheet.CustomerAdapter
```

Definitely a preference thing, but would be my preference.